### ISSUE TITLE
Add Identifier 'slug'

### ISSUE THREAD

**Issue by pdw207** [ResistanceCalendar/resistance-calendar-api#37]
Can we add a slug as an custom additional identifier to an event? We can use that for the event url.

**Comment by aaghevli** [comment]
Yes, that is a great idea. WRT to IDs in general, I was going to do:

hide the _id since its not OSDI compliant
add resistance_calendar:${_id} 
add resistance_calendar_slug: ${slugified(name)}

Does that sound like what you were thinking?

**Comment by pdw207** [comment]
Maybe `resistance_calendar_id` and  `resistance_calendar_slug`?
